### DISCUSSION TITLE
Allow non-file-visiting buffers?

### DISCUSSION THREAD

**Discussion by akurth** [alphapapa/activities.el#8] [Ideas]
Hey Adam,

I just found version 0.3 appearing in list-packages and immediately tried it out. I like the idea, thanks for starting this project!

Currently, when I try to save a window setup including non-file buffers (e.g. *scratch*), I get an error. I think, an "IDE-like" setup with treemacs on the left and some windows (say one or more code buffers and a *compilation* buffer) would be great to have as an activity setup. Do you think that would be feasible at all?

Cheers, Andreas.

**Comment by alphapapa** [comment]
Hi Andreas,

Thanks, I hope it is useful to you.

Please see the FAQ in the readme about unrestorable buffers and the bookmark system.  The `*scratch*` buffer should be restorable because it should always be open (unless the user killed it), but other non-file-backed buffers require bookmark support.
